config/src/main/java/com/typesafe/config/ConfigFactory.java

```diff
@@ -1090,6 +1090,63 @@ public static Config parseResourcesAnySyntax(String resourceBasename) {
         return parseResourcesAnySyntax(resourceBasename, ConfigParseOptions.defaults());
     }
 
+    /**
+     * Parse only any application overrides (those specified by config.{resource,file,url}), returning
```

Comment:
The word "override" already has a different meaning for `ConfigFactory.defaultOverrides` so I wonder what else we could call this ... the README uses the word "replacement" perhaps? `parseApplicationReplacement`?